<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>platformClass</class>
 <widget class="QMainWindow" name="platformClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>711</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>연구실 플랫폼 연습용</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/Deep Imaging Laboratory.bmp</normaloff>Resources/Deep Imaging Laboratory.bmp</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_ScreenHeight">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>600</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
        <item>
         <spacer name="horizontalSpacer_ScreenWidth">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(227, 227, 227);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>258</width>
            <height>155</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>258</width>
            <height>640</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Tool</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>60</y>
              <width>257</width>
              <height>28</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="topMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="minimumSize">
                <size>
                 <width>94</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>71</width>
                 <height>20</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(23, 168, 26);
font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
               </property>
               <property name="text">
                <string>Param3</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>3.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>257</width>
              <height>28</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="topMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>94</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>71</width>
                 <height>20</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(23, 168, 26);
font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
               </property>
               <property name="text">
                <string>Param1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="pushButton_run">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>99</y>
              <width>221</width>
              <height>25</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
            </property>
            <property name="text">
             <string>알고리즘 수행</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>257</width>
              <height>28</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="topMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="minimumSize">
                <size>
                 <width>94</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>71</width>
                 <height>20</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(23, 168, 26);
font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
               </property>
               <property name="text">
                <string>Param2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 10.5pt &quot;맑은 고딕&quot;;</string>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>3.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="platform.qrc"/>
 </resources>
 <connections/>
</ui>
